src/PAModel/Schemas/PcfControls/AuthConfigProperty.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerPlatform.Formulas.Tools.Schemas
+{
+    public class AuthConfigProperty
+    {
+        public int SectionIndex { get; set; }
```

Comment:
How many of these properties do we actually need? For ones that we aren't making use of, we could probably just remove them and rely on ExtensionData so it's less fragile w.r.t Studio changes

Reply:
As such currently no property is required now, when we do the default handling when the editorState.json is deleted we might require few. Will add those properties later on demand, for now I will remove all the properties.